Source/Orts.Simulation/Simulation/RollingStocks/MSTSDieselLocomotive.cs

```diff
@@ -968,15 +968,27 @@ public string GetDPDebugStatus()
             // Throttle
             status.AppendFormat("{0}\t", throttle);
             // Load
+            var avgLoadPercent = 0f;
+            var totalLoadW = DieselEngines.NumOfActiveEngines > 0 ? LocomotivePowerSupply.ElectricTrainSupplyPowerW : 0f;
+            var totalAvailablePowerW = 0f;
             foreach (var eng in DieselEngines.DEList)
-                status.AppendFormat("{0:F1}%\t", eng.LoadPercent);
+            {
+                totalLoadW += eng.CurrentDieselOutputPowerW <= 0f ? 0f : eng.OutputPowerW;
```

Comment:
In the same line, you use CurrentDieselOutputPowerW and OutputPowerW.
Is that normal?